app/models/grda_warehouse/service_history_service.rb

```diff
@@ -60,6 +60,10 @@ class GrdaWarehouse::ServiceHistoryService < GrdaWarehouseBase
     where(age: (18..Float::INFINITY))
   end
 
+  scope :bed_night, -> do
+    where(service_type: 200)
+  end
```

Comment:
Maybe this should go in `app/models/concerns/service_history_service_concern.rb`?